wire-ios-request-strategy/Sources/Payloads/Processing/Helpers/MLSEventProcessor.swift

```diff
@@ -27,7 +28,7 @@ protocol MLSEventProcessing {
 
 }
 
-final class MLSEventProcessor: MLSEventProcessing {
+struct MLSEventProcessor: MLSEventProcessing {
```

Comment:
Why do we change this to a struct? Is it related with the changes of this PR?

Reply:
It's neither related nor required here.
I usually try to fix issues when I come across them.
Let me know, if you prefer this change to be reverted.

Reply:
Let's keep it for now!

I think it would be beneficial for us to adopt a more structured approach in the future. When alterations like these are made, it would be helpful to have them included in a separate pull request, accompanied by a detailed explanation. This way, it becomes easier for the team to review and understand the context and rationale behind each change. Additionally, this method would improve our ability to track when and why specific modifications are implemented, enhancing overall transparency and efficiency in our workflow.